### DISCUSSION TITLE
RichTextEdit - Max Character Length

### DISCUSSION THREAD

**Discussion by holderbr** [Megabit/Blazorise#5219] [Q&A] [answered]
I'm using RichTextEdit for a project and my boss wants there to be a hard character limit. I know it doesn't support such a thing, but has anyone figured out, or has an idea for a potential work-a-round?

**Comment by David-Moreira** [comment, marked as answer]
Hello,

Have you tried setting up the `ValueChanged` handler, and whenever the `Value` exceeds the limit, you replace it with the maximum allowed value? This might possibly yield satisfactory results.

**Reply by holderbr** [reply]
Thank you for the suggestion. This does the trick. For some reason I didn't think you could pass values back to the editor, which was just foolish of me.
